Makefile

```
@@ -260,6 +260,7 @@ ios-simulator-arm: $(IOS_SIMULATOR_ARM) ## Build core-crypto-ffi for aarch64-app
 $(STAMPS)/ios: ios-device ios-simulator-arm
 	$(TOUCH_STAMP)
 
+ios-deps := $(RUST_SOURCES)
```

Shouldn't it be:

```suggestion
ios-deps := $(ios-device-deps) $(ios-simulator-arm-deps)
```

Granted it comes out to the same thing, but I think this version will be more maintainable in the event something changes.